M365/Check-MailboxExtendedProperty.ps1

```diff
@@ -0,0 +1,133 @@
+﻿# Copyright (c) Microsoft Corporation.
```

Comment:
We should probably change this script name to `Test-MailboxExtendedProperty`. We tend to follow the guidelines of approved verbs for PowerShell, at least for the public name of the scripts/functions.  

https://learn.microsoft.com/en-us/powershell/scripting/developer/cmdlet/approved-verbs-for-windows-powershell-commands?view=powershell-7.4

Reply:
Renamed to Test-